Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -135,8 +135,6 @@ theorem ofNat_zero : ofNat 0 = 0 :=
 theorem ofNat_succ (n) : ofNat (Nat.succ n) = oadd 0 n.succPNat 0 :=
   rfl
 
-attribute [eqns ofNat_zero ofNat_succ] ofNat
-
```

Comment:
There is a porting note on line 130 that refers to this line. I think we should have a tracking issue for "things that `eqns` tried to solve", and update all the existing porting notes near these deleted lines to refer to it.

Reply:
(https://github.com/leanprover/lean4/issues/2042 was originally intended to be that issue, but I didn't capture the intent quite well enough, and the solution fixed a different space of problems to the one I was trying to report)

Reply:
In fact, this one is used, so I'll just revert.